vault/static/vault/css/vault.css

```diff
@@ -118,3 +118,16 @@ table.dataTable > tbody > tr > td {
     color: #444;
     padding-left:3px;
 }
+
+.meta-content-size {
+    text-align: justify;
```

Comment:
We can use Bootstrap class `text-justify`

Reply:
Not implementing text justify